.github/workflows/vast.yaml

```diff
@@ -1096,3 +1096,43 @@ jobs:
       - name: Run unit tests
         working-directory: python
         run: poetry run pytest
+      - name: Build package
+        working-directory: python
+        run: poetry build
+
+  python-package:
+    needs:
+      - python
+      - determine-version
+    if: github.event_name == 'release'
+    name: Python Package
+    runs-on: ubuntu-latest
+    env:
+      DEBIAN_FRONTEND: noninteractive
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.ref }}
+      - name: Set up Python 3.10
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
```

Comment:
This is hard-coded, but above we're using a matrix. Does poetry upload packages per Python version, or do we always just use the latest version here?

Reply:
Yes, this is on purpose. We want to publish the package exactly once, not once per Python version.

Reply:
(Please resolve, if you do not disagree.)